### ISSUE TITLE
Update woodstox-core-asl dependency to latest fasterxml version

### ISSUE THREAD

**Issue by husayt** [FasterXML/jackson-jaxrs-xml-provider#12]
I have noticed, there is new fasterXml version woodstox-core-asl 5.0.0 coming to fruition soon.

Would be nice to update the usage here from old codehaus  version below to new one:

```
 <dependency>
      <groupId>org.codehaus.woodstox</groupId>
      <artifactId>woodstox-core-asl</artifactId>
      <version>4.1.2</version>
      <scope>test</scope>

```

**Comment by cowtowncoder** [comment]
Good point, thank you! I can start by upgrading to 4.4 (for 2.5.x), and then 2.6 can use 5.0.
There are no big changes in Woodstox so that should be safe upgrade, but it does change maven group id.

**Comment by husayt** [comment]
Yes, will be great to finally get whole stack in fasterxml namespace.

Thanks

**Comment by husayt** [comment]
Hi @cowtowncoder ,
just wanted to remind about this one. Hope this will help with Woodstox 4.4+ [Appengine issue](http://stackoverflow.com/questions/27208837/jackson-2-4-2-fails-on-appengine).

Thanks a lot

**Comment by cowtowncoder** [comment]
Jackson 2.6 and 2.7 are using Woodstox 5.0.1 aren't they?
